Makefile

```diff
@@ -14,14 +16,55 @@ access-mattermost:
 access-pagerduty:
 	make -C access/pagerduty
 
-.PHONY:
+.PHONY: access-example
 access-example:
 	go build -o build/access-example ./access/example
 
-.PHONY:
+# Run all tests
+.PHONY: test
+test:
+	go test -count 1 ./...
```

Comment:
nit: consider adding `-race`

Reply:
I did try this. Unfortunately it seems that there is actually a race in one of the tests, so I've reverted it for now. I will look into this in some more detail.